ports/hexl/portfile.cmake

```
@@ -0,0 +1,45 @@
+vcpkg_fail_port_install(ON_ARCH "x86" "arm" "arm64")
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO intel/hexl
+    REF c28943d89bb747fce93a8565f6213bf8414a5bd6
+    SHA512 9b9561bce128b55450a33ffaba660d369048dbffc736d26a4c3369dfa4a3b4f1250bee43ab2d307e32116d27d7e463008ad8ac96571130f0da04e3b82f1cecbb
+    HEAD_REF main
+)
+
+if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
+    set(HEXL_SHARED OFF)
+else()
+    set(HEXL_SHARED ON)
+endif()
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA
+    DISABLE_PARALLEL_CONFIGURE
+    OPTIONS
+        -DHEXL_DEBUG=OFF
+        -DHEXL_BENCHMARK=OFF
+        -DHEXL_EXPORT=ON
+        -DHEXL_COVERAGE=OFF
+        -DHEXL_TESTING=OFF
+        -DHEXL_SHARED_LIB=${HEXL_SHARED}
+        ${FEATURE_OPTIONS}
+)
+
+vcpkg_build_cmake(TARGET hexl LOGFILE_ROOT build)
+
+vcpkg_install_cmake()
+
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake)
```

```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake)
```
The same as above. See https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_fixup_cmake_targets.md